### PR TITLE
Embed Types

### LINKED ISSUE #247: EmbedType
I missunderstood requirements for embedTypes and didn't take enough care to previous implementation of this functionality. But PR #224 break this functionnality.

Before #224, we simply needed to create a generator file with only the "global" configuration.
Now, you need to create a full generator file with at least Edit and New builders. Doing that, corresponding controllers will be created too (that wasn't the case before the PR).

That should be fixed.

### PR DESCRIPTION
**Pull request by sescandell**
FIX #247

 - [x] Generate embed types from generators
 - [x] Auto-inject embed types

### CODE REVIEW

File: Generator/DoctrineODMGenerator.php [hunk 1]
```diff
@@ -47,6 +47,8 @@ protected function doBuild()
         );
         $generator->setBaseGeneratorName($this->getBaseGeneratorName());
 
+        $this->generateEmbedTypes($generator);
```
Comment (bobvandevijver) [inline Generator/DoctrineODMGenerator.php]: I miss the `typeBuilderNamespace` here

File: Generator/PropelGenerator.php [hunk 2]
```diff
@@ -49,6 +49,8 @@ protected function doBuild()
         $generator->setColumnClass('Admingenerator\GeneratorBundle\Generator\PropelColumn');
         $generator->setBaseGeneratorName($this->getBaseGeneratorName());
 
+        $this->generateEmbedTypes($generator);
```
Comment (bobvandevijver) [inline Generator/PropelGenerator.php]: And here

**Comment by sescandell** [comment]
@bobvandevijver yes... commit was pending :)

**Comment by bobvandevijver** [comment]
Well, I was to fast. That's new :D

**Comment by sescandell** [comment]
Well... I probably was tired... it's already working :)

PR cancelled

**Pull request closed without merging**